static/css/style.css

```diff
@@ -183,6 +183,29 @@ figure{
     display: inherit;
 }
 
+/* Accessibility Page */
+.card-title {
+    font-size: xx-large;
+    text-align: center;
+    padding: 1.5em 0;
+}
+
+.card-text {
+    font-family: "Roboto Condensed", sans-serif;
+    font-size: 16px;
+    padding: 1rem 15%;
+    text-align: center;
+}
+
+.img-fluid {
+    width: 250px;
+}
+
+.text-center {
+    text-align: center !important;
+    background-color: white;
```

Comment:
Can this be changed to `var(--p1-light)` instead of using the keyword of `white` to suit the rest of the CSS file? 😀

We have all our color variables mapped to the `:root` pseudo element, allowing us to reuse those variables, and if needed, change them in one place and then filtering that one change wherever that variable is declared.